packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.spec.js

```diff
@@ -9,6 +9,7 @@ import { default as Task } from '@plugins/tasks/SurveyTask'
 import FilterStatus from './FilterStatus'
 import Characteristics from '../Characteristics'
 import FilterButton from '../Characteristics/components/FilterButton'
+import { expect } from 'chai'
```

Comment:
I think VSCode autocompletes/autoinserts this?

Reply:
It does. I didn't catch that it had here.